LocalSettings.php

```diff
@@ -2394,6 +2394,12 @@
 		'wisdomsandboxwiki' => '_blank',
 		'yacresourceswiki' => '_blank',
 	),
+	
+	// Allow External Images
+	'wgAllowExternalImages' => array (
+		'default' => false,
+		'westmarcheswiki' => true
```

Comment:
I believe this should have a comma after the false (based on the above array) however I don't believe it's acruallly a syntax error.